### DISCUSSION TITLE
Next unknown build errors

### DISCUSSION THREAD

**Discussion by meilechwieder** [vercel/next.js#16190] [Help]
# Bug report
I am unable to build my next app.
## Describe the bug
When I build locally on my PC, I have no problems. When I build on heroku or on vercel, I get the errors below saying "cant resolve module xxxxxx". I was able to build in the past with vercel. I do not no what I did wrong since my last commit that is causing those errors. I am currently running next version 9.5.1, but I also had errors when I downgraded to version 9.3.3
## To Reproduce
Failed to compile.
remote:
remote: ./components/layout/landing/HomePageTabs.js
remote: Module not found: Can't resolve '../../products/productDetails/ProductAccessories' in '/tmp/build_4be1adef/components/layout/landing'
remote:
remote:
remote: > Build error occurred
remote: Error: > Build failed because of webpack errors
remote:     at build (/tmp/build_4be1adef/node_modules/next/dist/build/index.js:13:900)
remote: npm ERR! code ELIFECYCLE
remote: npm ERR! errno 1
remote: npm ERR! optics_force@0.1.0 build: `next build`
remote: npm ERR! Exit status 1
remote: npm ERR!
remote: npm ERR! Failed at the optics_force@0.1.0 build script.
remote: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
remote: 
remote: npm ERR! A complete log of this run can be found in:
remote: npm ERR!     /tmp/npmcache.osZbj/_logs/2020-08-14T15_00_22_594Z-debug.log
remote: 
remote: -----> Build failed

- Version of Next.js: 9.5.1
- Local Version of Node: 14.4

**Comment by daveteu** [comment]
likely you will see the error locally once you remove your nose modules folder then reinstall, and you may find you had some path mixed up (typo or whatever). happens a lot to anyone.

**Comment by IRediTOTO** [comment]
It said `Module not found: Can't resolve '../../products/productDetails/ProductAccessories' in '/tmp/build_4be1adef/components/layout/landing'`
This is your error, not from Next

**Comment by IRediTOTO** [comment]
Or just need delete the folder `.next` and run again.

**Comment by timneutkens** [comment]
This error generally happens when you import a file but the casing is wrong, on case-insensitive file systems (which both MacOS and Windows have by default) the import would resolve fine, however on case-sensitive filesystems (which most deployment services have) it would lead to a problem accessing the file
